BAZProjectC1/Components/CarosuelMenu/CarosuelMenuExtension.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CarosuelMenuExtension.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 23/09/22.
+//
+
+import Foundation
+import UIKit
+
+fileprivate let PADDING_SCREEN_PIXEL: CGFloat = 16
+fileprivate let FONT_PONTSIZE_PIXEL: Int = 8
+fileprivate let ARRAY_HELPER_SIZE: Int = 1
```

Comment:
Bien hecho miviendo estos valores a constantes.

Solo recuerda que la convención de nombrado de propiedades en Swift siguen el lowerCammelCase, independientemente de que sean constantes.

- [Convención Constantes](https://github.com/raywenderlich/swift-style-guide#constants)
- [Convención de nombrado](https://github.com/raywenderlich/swift-style-guide#naming)

La convención que estas usando es más bien de otros lenguajes como Javascript, Java o Kotlin

Si necesitas especificar que son constantes podrías encapsularlas en una estructura llamada `Constants`, por ejemplo:
```suggestion
struct Constants {
    fileprivate let paddingScreenPixel: CGFloat = 16 
    fileprivate let fontPointSizePixel: Int = 8
    fileprivate let arrayHelperSize: Int = 1
}
```

Incluso esta estructura podría estar declarada dentro de tu clase, por ejemplo:

```
class CarosuelMenu: UIView {
    (...)

    struct Constants {
        fileprivate let paddingScreenPixel: CGFloat = 16 
        fileprivate let fontPointSizePixel: Int = 8
        fileprivate let arrayHelperSize: Int = 1
    }

    (...)
}
```